### PR TITLE
 Add Belen's data

### PR DESCRIPTION
**Pull request by Belen96x**
Added my data.

### CODE REVIEW

**Comment by Javanochka** [comment]
Please, resolve your own merge conflict :)

**Comment by Belen96x** [comment]
I think now its resolved!

File: class01-git/facts.py [hunk 1]
```diff
@@ -11,10 +11,9 @@
...
-    Person(github_login='tunji17', first_name='Oyetunji', last_name='Abioye',
-            facts=['I am not Nigerian', 'I am a Ms student', 'I can fly']),
-    Person(github_login="robin", first_name="Tweety", last_name="Bird",
-           facts=["I have arms", "I can fly", "I eat worms"]),
+    Person(github_login='Belen96x', first_name='Belen', last_name='Saavedra', facts=['I am a linguist', 'I travel with my cat', 'I can fly']),
+    Person(github_login='tunji17', first_name='Oyetunji', last_name='Abioye', facts=['I am not Nigerian', 'I am a Ms student', 'I can fly']),
+    Person(github_login="robin", first_name="Tweety", last_name="Bird", facts=["I have arms", "I can fly", "I eat worms"])
 ]
 # facts are numerated from 0
 answers = {'example': 1,
```
Comment (Javanochka) [inline class01-git/facts.py:19]: Please, add a correct answer for yourself here :)

Comment (Belen96x) [reply class01-git/facts.py:19]: I think now it's done!

**Comment by Belen96x** [comment]
Changes applied and no merge conflicts

**Comment by Javanochka** [comment]
Good work!

**Pull request closed without merging**